### PR TITLE
Code Formatting Units V1

### PR DESCRIPTION
**Pull request by MrRowey**

### CODE REVIEW

File: units/UAB3101/UAB3101_script.lua [hunk 1]
```diff
@@ -10,22 +10,32 @@
...
     OnIntelEnabled = function(self, intel)
+        local rotator = self.Rotator1
+        local trash = self.Trash
+
         ARadarUnit.OnIntelEnabled(self, intel)
-        if not self.Rotator1 then
-            self.Rotator1 = CreateRotator(self, 'B01', 'y')
-            self.Trash:Add(self.Rotator1)
+        if not rotator then
+            rotator= CreateRotator(self, 'B01', 'y')
```
Comment (speed2CZ) [inline units/UAB3101/UAB3101_script.lua:33]: This logic is broken, the rotator will never get saved on the unit. This applies to all similar changes you did in this PR

**Review by lL1l1 — CHANGES_REQUESTED** [review]
See https://github.com/MrRowey/fa/pull/2 for the suggested changes. It fixes some critical typos, fixes the broken upvalues, fixes some animation bugs, adds new upvalues, and fixes some whitespace.

**Review by lL1l1 — APPROVED** [review]

**Comment by MrRowey** [comment]
@Garanas this is ready for review

**Comment by lL1l1** [comment]
Testing done:
Diff the branch with fafdevelop, then spawn in all the units that appeared in the diff:
<details> <summary> Spawn command: </summary>

```lua
CreateUnitAtMouse('daa0206', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('dal0310', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('dea0202', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('delk002', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('dra0202', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('drlk001', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('dslk004', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uaa0104', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uaa0203', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uaa0304', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uaa0310', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab0303', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab0304', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1102', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1103', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1104', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1106', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1201', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1301', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1302', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab1303', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab2109', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab2204', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab2205', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab3101', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab3104', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab3201', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab4201', 1,    0.00,    0.00, -0.71822)
CreateUnitAtMouse('uab4202', 1,    0.00,    0.00, -0.71822)
```
</details>

**Pull request merged**